setup.cfg

```
@@ -1,4 +1,66 @@
-# This file should be used as the pep8/pycodestyle and autopep8 config
+###############################################################################
+# Setuptools
+###############################################################################
+
+[metadata]
+name = WebAppDIRAC
+description = WebAppDIRAC is a portal for the DIRAC software.
+long_description = file: README.rst
+url = https://github.com/DIRACGrid/WebAppDIRAC/
+license = GPL-3.0-only
+license_files = LICENSE
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+    Topic :: System :: Distributed Computing
+
+[options]
+python_requires = >=3.8
+package_dir=
+    =src
+packages = find:
+
+install_requires =
+    dirac
```

I guess this should have a version associated with it as I guess this tends to be tightly coupled to DIRAC?
```suggestion
    dirac ~=7.3.0a0
```